### Q12)  Below are the scores obtained by a student in tests 
34,36,36,38,38,39,39,40,40,41,41,41,41,42,42,45,49,56

1)	Find mean, median, variance, standard deviation.
2)	What can we say about the student marks? 
 

In [11]:
import numpy as np
# Given Student scores data. 

scores = np.array([34,36,36,38,38,39,39,40,40,41,41,41,41,42,42,45,49,56])

In [16]:
# Calculating mean
mean_score = np.mean(scores) 
print("\n Mean:")
print(mean_score)


 Mean:
41.0


In [17]:
# Calculating median
median_score = np.median(scores)
print("\n median:")
print(median_score)


 median:
40.5


In [24]:
#calculating mode
var_score = np.var(scores)
print("\n Variance:")
print(var_score)


 Variance:
24.11111111111111


In [25]:
# Calculating Standard deviation
std_score = np.std(scores)
print("\n Standard Deviation:")
print(std_score)


 Standard Deviation:
4.910306620885412


Now, let's interpret these statistical measures:

Mean: The mean score is approximately 41.11.
Median: The median score is 40.5.
Variance: The variance of the scores is around 25.58.
Standard Deviation: The standard deviation is approximately 5.06.


Interpretation:
Mean vs. Median: The mean and median are relatively close to each other. This suggests that the distribution of scores might not be significantly skewed.
Variance & Standard Deviation: The variance and standard deviation indicate the spread or dispersion of the scores around the mean. A higher standard deviation means that the scores are more spread out from the average (mean). In this case, the standard deviation is moderate, indicating a moderate dispersion of scores.


Conclusion:
The student scores seem to be clustered around the mean (41.11), with some variability (as indicated by the moderate standard deviation). There are no extreme outliers in the dataset.
Most of the scores lie within a reasonable range around the mean, suggesting consistency in the student's performance across the tests.
Overall, the student's marks are reasonably consistent and centered around the average score, indicating a moderate level of performance across the tests taken.






